frontend/src/libs/components/prompt-generation/libs/components/prompt-card/prompt-card.tsx

```diff
@@ -16,9 +16,9 @@ const PromptCard: React.FC<Properties> = ({ category, text, isGenerating }) => (
       <span className={styles.categoryText}>{category}</span>
     </div>
     <div className={styles.prompt}>
-      <DotLoader isLoading={isGenerating}>
+      <Loader isLoading={isGenerating} hasOverlay={false} loaderType="dots">
```

Comment:
And same about enum usage
```suggestion
      <Loader isLoading={isGenerating} loaderType="dots">
```